modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/side_panel/SidePanel.js

```diff
@@ -382,21 +385,29 @@ export default class SidePanel extends React.Component {
 						/>
 					)}
 
-					<ClayButton
-						aria-label={Liferay.Language.get('close')}
-						className={classNames(
-							'fds-side-panel-close',
-							this.state.closeButtonStyle === 'simple' &&
-								'fds-side-panel-close-simple',
-							this.state.closeButtonStyle === 'menu' &&
-								'fds-side-panel-close-menu'
-						)}
-						displayType="monospaced"
-						onClick={() => this.close()}
-						title={Liferay.Language.get('close')}
-					>
-						<ClayIcon symbol="times" />
-					</ClayButton>
+					{this.state.title && (
```

Comment:
Apply the same mechanism we are using for the [Modal](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/modal/Modal.js#L144)

Reply:
That's the wrong Modal, that we want to get rid of :slightly_smiling_face:  The real Modal in [the one in frontend-js.web](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/modal/Modal.js#L219-L237). There we have the explicit `disableHeader` flag. That is more flexible than coupling `title` with header, as it allows `headerHTML` as well.

Reply:
So we'd like to homogeneize sidepanel and modal APIs as much as possible. Still I don't see the need to enable `headerHTML` in the sidepanel for now

Reply:
Yeah, I didn't mean we should implement `headerHTML`, at least not for now, just highlighting the advantage of an explicit flag. It could also help resolve mandatory title concern in https://github.com/liferay-frontend/liferay-portal/pull/4176#issue-2283556401